predicate-builder-service/README.md

```diff
@@ -0,0 +1 @@
+# TODO:
```

Comment:
Can you document the service please?
Also, it would be pretty useful to have an example request, maybe a curl, and show the result for that given request.